### ISSUE TITLE
Rule node config ui?

### ISSUE THREAD

**Issue by Philip2809** [thingsboard/thingsboard#10501]
**Component**

 * Rule Engine
 * Installation
 * 
**Description**

Is https://github.com/thingsboard/thingsboard-rule-config-ui still used today? Is it possible to configure how the rulenode-core-config.js works?

**Comment by devaskim** [comment]
You need this repo [https://github.com/thingsboard/thingsboard-rule-config-ui-ngx](https://github.com/thingsboard/thingsboard-rule-config-ui-ngx)

... and use issue's title ;)

**Comment by Philip2809** [comment]
I don't exactly understand what you mean, you say i need both? does it work with latest 3.6.3?

Also thank you for answering all my silly questions :D

**Comment by devaskim** [comment]
>  you say i need both?

No, only `[thingsboard-rule-config-ui-ngx](https://github.com/thingsboard/thingsboard-rule-config-ui-ngx)`

> does it work with latest 3.6.3?

Github, like other Git hosters, has useful feature called Commit history (click on `Commits` at right top corner on the main repo page) 
![Скриншот 05-04-2024 113701](https://github.com/thingsboard/thingsboard/assets/86909129/81959090-91e1-40c5-a867-073f70ddea44)

**Comment by Philip2809** [comment]
Yea, found that in the end, but didnt really understand how to get it working, but now I understand from someone elses comment, I use the proxy.config.json in the thingsboard ui-ngx right?

**Comment by mykhailokornieiev** [comment]
We're letting you know that this issue, labeled as "help wanted", has been open for a while. This label is used to seek support from the GitHub community, especially for issues that fall outside the scope of the ThingsBoard support team's expertise. Unfortunately, there hasn't been any community input or progress on this issue over the past three months. We're closing this ticket to keep our repository well-maintained and focused on active issues. We appreciate your patience and understanding.
